api/v1alpha1/backend_types.go

```diff
@@ -161,7 +161,6 @@ type BackendSpec struct {
 
 // BackendTLSSettings holds the TLS settings for the backend.
 // +kubebuilder:validation:XValidation:message="must not contain both CACertificateRefs and WellKnownCACertificates",rule="!(has(self.caCertificateRefs) && size(self.caCertificateRefs) > 0 && has(self.wellKnownCACertificates) && self.wellKnownCACertificates != \"\")"
-// +kubebuilder:validation:XValidation:message="must specify either CACertificateRefs or WellKnownCACertificates",rule="(has(self.caCertificateRefs) && size(self.caCertificateRefs) > 0 || has(self.wellKnownCACertificates) && self.wellKnownCACertificates != \"\")"
```

Comment:
This should be checked and not allowed: WellKnownCACertificates == nil && caCertificateRefs == nil && (InsecureSkipVerify == nil || InsecureSkipVerify == false)

Reply:
> InsecureSkipVerify == true && (WellKnownCACertificates != nil || caCertificateRefs != nil)

that's reasonable

> (InsecureSkipVerify == nil || InsecureSkipVerify == false) && WellKnownCACertificates == nil && caCertificateRefs == nil

`InsecureSkipVerify == false && WellKnownCACertificates == nil && caCertificateRefs == nil` I think this case makes sense, doesn't it?